gui/services/migrations/0003_add_ntlmv1_auth.py

```diff
@@ -16,6 +16,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='cifs',
             name='cifs_srv_ntlmv1_auth',
-            field=models.BooleanField(default=False, help_text='This parameter determines whether or not smbd(8) will attempt toauthenticate users using the NTLMv1 encrypted password response', verbose_name='NTLMv1 auth'),
+            field=models.BooleanField(default=False, help_text='This parameter determines whether or not smbd(8) will attempt to authenticate users using the NTLMv1 encrypted password response. Not recommended because the encryption is weak and easily defeated.', verbose_name='NTLMv1 auth'),
```

Comment:
Suggestion: 'Off by default. When set, smbd(8) attempts to authenticate users with the insecure and vulnerable NTLMv1 encryption. This setting allows backward compatibility with older versions of Windows, but is not recommended and should not be used on untrusted networks.'

Reply:
I'm not too bothered about the wording. I just think it needs to be warned against.

Feel free to close this PR and use whatever wording the project maintainers prefer.

I'm new to this project, I guess maybe filing an issue is better in these cases?  So wording can be iterated?

Reply:
In general, a bug report in https://redmine.ixsystems.com/projects/freenas is best, because it is tracked.